CMakeLists.txt

```diff
@@ -283,7 +285,10 @@ if (ANDROID)
 endif()
 
 if (CYGWIN)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-exceptions -fno-rtti")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
+    if (NOT FILAMENT_ENABLE_CPP_EXCEPTIONS)
+        set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS} -fno-exceptions")
```

Comment:
Shouldn't this be `set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-exceptions")` instead of `CMAKE_CXX_FLAGS_RELEASE`?

Reply:
It was, but then a copy-paste came...

Reply:
ddca9a10e8c11727338d57d6d2f9ce930f195904